src/screens/ImportOrWatchWalletSheet.tsx

```diff
@@ -60,7 +62,7 @@ export const ImportSeedPhraseSheet: React.FC = () => {
   return (
     <Box
       height={{
-        custom: contentHeight,
+        custom: deviceUtils.dimensions.height,
```

Comment:
Would be better to use a hook like useDimensions. It is reactive to orientation changes. We do not support any more orientations than portrait and changing window size, but still we could do that.